### ISSUE TITLE
problems compiling FEHM with intel visual fortran 

### ISSUE THREAD

**Issue by jbensabat** [lanl/FEHM#22]
Hello 
I am trying to build FEHM with intel Visual fortran under windows.
I get a large number of errors of the same type around this portion of code 

c     add coding to save upwind position
               if(iad.le.iad_up) then
                  fid=0.5
                  axyd=t8(neighc)
                  if(axyd.lt.0.0) fid=dnwgt
                  if(axyd.gt.0.0) fid=upwgt
                  t9(neighc)=fid
c     
                  call setbit(nbits,neighc,upwind_l(iz4m1),fid)
c     
               else
                  if(bit(nbits,neighc,upwind_l(iz4m1))) then 
                     t9(neighc)=1.0
                  else
                     t9(neighc)=0.0
                  endif
               endif
the error arises because subroutine setbit accepts integers only but the array upwind_l if real.
I saw that in some routines any reference to setbit was commented 
Is it ok to do so ?
thanks 
jac

**Comment by daniellivingston** [comment]
Hi Jac, try commenting out `bit` and `setbit`:

> I would say that if the new Intel compiler does not work with 'bit' and 'setbit' we comment out those. Macro 'itup' will be essentially disabled but very few of us have used it. I think no test cases do.

**Comment by jbensabat** [comment]
Hi and thank you for your support 
I resolved this issue by checking out the flag "Check Routine Interfaces" in the "Diagnostics" option of the fortran workspace properties. I was able then to build FEHM. There are two types of issues: 1) related to the setbit subroutine - the dummy arguments are all integer (4 bytes) while the third argument in the calling is of type real*8. even if compilation goes thru I am not sure that results could be reliable; 2) there are many cases where the size of dummy arrays are larger than the ones in the calling sequence - but this is not a problem.

thanks
jac
